### DISCUSSION TITLE
Cannot get admitted

### DISCUSSION THREAD

**Discussion by 1337adde** [community/community#53287] [GitHub Education]
I have tried several times to apply for GitHub Education, but cannot get admitted. I get different error messages almost every time I make an application, but they always contain one or several of these: 

- You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

- Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?

- The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.

- You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

- Please upload proof of your current school affiliation

I get different reasons every time, even if I do the application in the exact same way...

I am applying with a dated certificate of enrollment that states that my education is at a distance. How ever, I also tried applying using my university's VPN, but that does not work either.

**Comment by samus-aran** [comment]
Hi @1337adde 👋 

How did you get on? Did you do anything different in any later submissions?

**Reply by 1337adde** [reply]
I actually got admitted after applying for 15-20 more times, trying different options in the drop down menu everything. I don't know which one that actually worked out in the end.
